test/distributed/_composable/fully_shard/test_fully_shard_compile.py

```diff
@@ -39,9 +39,11 @@ def test_compile(self):
                     ShardingStrategy.FULL_SHARD,
                     ShardingStrategy.SHARD_GRAD_OP,
                     ShardingStrategy.NO_SHARD,
+                    ShardingStrategy.HYBRID_SHARD,
+                    ShardingStrategy._HYBRID_SHARD_ZERO2,
                 ],
                 "skip_fsdp_guards": [True, False],
-                "act_checkpoint": [True],
+                "act_checkpoint": [True, False],
```

Comment:
As commented on in the previous PR, I think this change should be done in that PR.